.bazelrc

```diff
@@ -6,13 +6,16 @@ build --enable_platform_specific_config
 # Verbose failure logs when something goes wrong.
 build --verbose_failures
 
+# Build with clang (rather than the default of gcc) since clang compiles
+# templates faster and produces better error messages.
+build --repo_env=CC=clang
+
 # Use Clang as a default compiler on Windows.
-# TODO: use Clang as a default compiler on every platform.
+# TODO(xander): Does repo_env above cover the windows case?
 build:windows --compiler="clang-cl"
 
-# Compiler options.
-build:gcc --action_env=CC=gcc
-build:clang --action_env=CC=clang
+# Make it easy to use gcc.
+build:gcc --repo_env=CC=gcc
```

Comment:
Can you please leave a comment why we have to use `--repo_env` instead of `--action_env`?

Reply:
Done.